src/AppInstallerCommonCore/Public/winget/Settings.h

```diff
@@ -76,6 +78,8 @@ namespace AppInstaller::Settings
         constexpr static StreamDefinition BackupUserSettings{ Type::UserFile, "settings.json.backup"sv };
         // The admin settings.
         constexpr static StreamDefinition AdminSettings{ Type::Secure, "admin_settings"sv };
+        // Trusted remote settings.
+        constexpr static StreamDefinition TrustedRemoteSettings{ Type::TrustedRemote, "trusted_remote_settings.msix"sv };

```

Comment:
I was imagining that a `StreamDefinition` for this would be more like:
```
constexpr static StreamDefinition Name { Type::TrustedRemote, "package relative file path"sv };
```
Thus removing the need for `TrustedRemoteSettings.h|cpp`.

The well known `StreamDefinition` for `MSStoreSoureCertificatePinningConfiguration.json` (not sure of actual file name) could be here, and the code loading it would construct new `StreamDefinition`s for each file within that.  So:
```
"RemoteSettingFileCertificate":"cert.cer"
```
   becomes
```
std::string certFilePath = remoteSettingsFileCertificateNode.value();
StreamDefinition certStreamDef{ Type::TrustedRemote, certFilePath };
```
With `certFilePath == "cert.cer"`